src/app/services/bigquery.service.ts

```diff
@@ -196,6 +197,51 @@ export class BigQueryService {
     });
   }
 
+  NormalizeRows(rows: Array<Object>, normalized_cols: Array<Object>, stats: Map<String, ColumnStat>) {
```

Comment:
nit: For consistency `lowerCamelCase` for both the method and parameter names.

Reply:
Done.